proto/felixbackend.proto

```diff
@@ -223,6 +227,21 @@ message Rule {
   repeated string not_src_named_port_ip_set_ids = 112;
   repeated string not_dst_named_port_ip_set_ids = 113;
 
+  // These fields pass through the original selectors from the v3 datamodel unmodified as required
+  // for the policy sync API.
+  string original_src_selector = 114;
+  string original_dst_selector = 115;
+  string original_src_namespace_selector = 116;
+  string original_dst_namespace_selector = 117;
+  string original_not_src_selector = 118;
+  string original_not_dst_selector = 119;
+
+  // Pass through of the v3 datamodel service account match criteria.
+  string src_service_account_selector = 120;
+  string dst_service_account_selector = 121;
+  repeated string src_service_account_names = 122;
+  repeated string dst_service_account_names = 123;
```

Comment:
I think this would be a little nicer if we used Protobuf messages to structure this data. If feels right now like long names are doing the work that a `Message` should.

E.g.
```
message ServiceAccountSelector {
  string label_selector = 1;
  repeated string names = 2;
}
```

It makes it easier to write reuseable handling code this way.